### PR TITLE
Homework Page done with dummy data

### PR DESCRIPTION
**Pull request by abhishek2021005**

### CODE REVIEW

File: packages/alea-frontend/lang/en.ts [hunk 1]
```diff
@@ -145,6 +146,12 @@ export const en = {
       'is a demo quiz - so that you can test your hard/software (you will need a recent chrome or firefox browser) and see the format. In particular, you should be able to read the Math in the travelling salesperson problem.',
     practiceProblems: 'Practice Problems',
   },
+  homework: {
+    homeworkDashboard: 'Homework Dashboard',
+    previousHomeworks: 'Previous Homeworks',
+    onTimeWarning:
```
Comment (abchugh) [inline packages/alea-frontend/lang/en.ts:152]: rename this variable appropriately.

File: packages/alea-frontend/lang/en.ts [hunk 2]
```diff
@@ -145,6 +146,12 @@ export const en = {
       'is a demo quiz - so that you can test your hard/software (you will need a recent chrome or firefox browser) and see the format. In particular, you should be able to read the Math in the travelling salesperson problem.',
     practiceProblems: 'Practice Problems',
   },
+  homework: {
+    homeworkDashboard: 'Homework Dashboard',
+    previousHomeworks: 'Previous Homeworks',
```
Comment (abchugh) [inline packages/alea-frontend/lang/en.ts:151]: previousHomework to  "Given homeworks"

File: packages/alea-frontend/pages/course-home/[courseId].tsx [hunk 3]
```diff
@@ -203,6 +205,10 @@ const CourseHomePage: NextPage = () => {
               <QuizIcon fontSize="large" />
             </CourseComponentLink>
           )}
+          <CourseComponentLink href={`/homework/${courseId}`}>
```
Comment (abchugh) [inline packages/alea-frontend/pages/course-home/[courseId].tsx:208]: Let's hide this in production till the feature is ready:
``` 
process.env.NEXT_PUBLIC_SITE_VERSION !== 'production' && 
<CourseComponentLink href={`/homework/${courseId}`}>{t.homeworks}&nbsp;
            <AssignmentTurnedInIcon fontSize="large" />
          </CourseComponentLink>
```

File: packages/alea-frontend/pages/homework/[id].tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+import { useContext, useEffect, useState } from 'react';
+import HomeworkPerformanceTable from '../../components/HomeworkPerformanceTable';
+import { getLocaleObject } from '../../lang/utils';
+import MainLayout from '../../layouts/MainLayout';
+import { CourseHeader } from '../course-home/[courseId]';
+import { ForceFauLogin } from '../../components/ForceFAULogin';
+
+const HomeworkPage: NextPage = () => {
+  const router = useRouter();
+
```
Comment (abchugh) [inline packages/alea-frontend/pages/homework/[id].tsx:16]: Try to make use of empty lines carefully. Use them to separate our logically separate parts of code. Don't add space indiscriminately.

File: packages/alea-frontend/components/HomeworkPerformanceTable.tsx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+import Link from 'next/link';
+
+import { getLocaleObject } from '../lang/utils';
+import { useRouter } from 'next/router';
+
+function HomeworkPerformanceTable({ courseId }: { courseId: string }) {
+  const { homeworkPerformanceTable: t, homework: tHW } = getLocaleObject(useRouter());
+
+  const homeworkData = [
```
Comment (abchugh) [inline packages/alea-frontend/components/HomeworkPerformanceTable.tsx:23]: constants in upper snake case.
homeworkData --> HOMEWORK_DATA

**Review by abchugh — APPROVED** [review]
Good work. Only minor comments. Please merge after resolving.

File: packages/alea-frontend/lang/en.ts (hunk 1 above)
Comment (abhishek2021005) [reply packages/alea-frontend/lang/en.ts]: Fixed!!

File: packages/alea-frontend/lang/en.ts (hunk 2 above)
Comment (abhishek2021005) [reply packages/alea-frontend/lang/en.ts]: Done!

File: packages/alea-frontend/pages/course-home/[courseId].tsx (hunk 3 above)
Comment (abhishek2021005) [reply packages/alea-frontend/pages/course-home/[courseId].tsx]: Fixed !!

File: packages/alea-frontend/pages/homework/[id].tsx (hunk 4 above)
Comment (abhishek2021005) [reply packages/alea-frontend/pages/homework/[id].tsx]: Noted, will take care in upcoming PRs.

File: packages/alea-frontend/components/HomeworkPerformanceTable.tsx (hunk 5 above)
Comment (abhishek2021005) [reply packages/alea-frontend/components/HomeworkPerformanceTable.tsx]: Resolved.

**Pull request merged**